### PR TITLE
Updated ReadMe for Windows

### PR DESCRIPTION
**Pull request by drlukeangel**
add set config for windows

### CODE REVIEW

File: docs/user/README.md [hunk 1]
```diff
@@ -35,7 +35,10 @@ You may also try removing any unused data left by the Docker engine - e.g.,
 After [creating a cluster](#creating-a-cluster), you can use [kubectl][kubectl]
 to interact with it by using the configuration file generated by `kind`:
 ```
-export KUBECONFIG="$(kind get kubeconfig-path)"
+Mac/Linux
+  export KUBECONFIG="$(kind get kubeconfig-path)"
```
Comment (neolit123) [inline docs/user/README.md]: this indent here has two spaces while the one bellow has one space.

File: docs/user/README.md [hunk 2]
```diff
@@ -35,7 +35,10 @@ You may also try removing any unused data left by the Docker engine - e.g.,
...
+Windows Powershell
+ $env:KUBECONFIG="$(kind get kubeconfig-path)"
```
Comment (neolit123) [inline docs/user/README.md]: powershell is not really required for the sake of using kind or setting env variables.

for the regular cmd shell we could do this:
```
for /f "delims=*" %i in ('kind get kubeconfig-path') do @set KUBECONFIG="%i"
```
which is far from pretty.

the alternative which gets my vote is:
```
kind get kubeconfig-path > kindpath && set /p KUBECONFIG=<kindpath && del kindpath
```

**Comment by alejandrox1** [comment]
@drlukeangel have you seen the comments to your PR?

**Comment by neolit123** [comment]
@drlukeangel 
i'm going to close this PR and take ownership of this problem by sending docs and CLI updates related to windows.

thanks.
/close

**Pull request closed without merging**